packages/syft/src/syft/service/request/request.py

```diff
@@ -603,7 +652,7 @@ def _run(
         try:
             obj = self.linked_obj.resolve_with_context(context)
             if obj.is_err():
-                return SyftError(message=obj.err())
+                return Err(SyftError(message=obj.err()))
```

Comment:
@teo-milea there is a bit inconsistent use of Err and SyftError at request and Action store. Resorting to still use Result library over SyftError to prevent breaking existing functionality. We need to refactor this a bit and make their use consistent all across the codebase.